errors.py

```diff
@@ -0,0 +1,15 @@
+
+class MalformedMoYaml(Exception):
```

Comment:
I should have caught this in the initial code, but Exception already takes a value, or message, by default, so no extra code is needed in it.

```
class MalformedMoYaml(Exception):
    pass

...

raise MalformedMoYaml('No heroku apps defined for this project')
```